app/src/pages/inside/common/infoLine/parentInfo/duration/durationTooltip/durationTooltip.jsx

```diff
@@ -15,16 +15,97 @@
  */
 
 import classNames from 'classnames/bind';
-import { FormattedMessage } from 'react-intl';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import { approximateTimeFormat, dateFormat, getDuration, getApproximateTime } from 'common/utils';
+import {
+  isInProgress,
+  isStopped,
+  isInterrupted,
+  isSkipped,
+  messages,
+} from 'pages/inside/common/durationBlock/durationBlock';
 import styles from './durationTooltip.scss';
 
 const cx = classNames.bind(styles);
 
-export const DurationTooltip = () => (
-  <div className={cx('duration-tooltip')}>
-    <FormattedMessage
-      id="DurationTooltip.message"
-      defaultMessage="Duration is interval between first child starts and last child ends. But if child run in parallel, end time is a time of longest child, in this case duration will not be equal to child duration sum."
-    />
-  </div>
-);
+export const DurationTooltip = injectIntl(({ intl, status, timing, type }) => {
```

Comment:
We usually use useIntl hook for that purpose.

Reply:
fixed ✅